realm/realm-jni/build.gradle

```diff
@@ -81,6 +89,34 @@ if (project.clang instanceof String) {
 if (project.forceDownloadCore instanceof String) {
     ext.forceDownloadCore = Boolean.parseBoolean(project.forceDownloadCore)
 }
+if (project.stripSymbols instanceof String) {
```

Comment:
but `project.stripSymbols` seems always boolean since we define `ext.stripSymbols`.
Is that true?